chat-server/src/app.ts

```diff
@@ -93,5 +93,10 @@ export const setupApp = async () => {
   });
   app.use(errorHandler);
 
-  return app;
+  return Object.assign(app, {
+    async close() {
+      await mongodb.close();
+      await store.close();
+    },
```

Comment:
why separate `mongodb` and `store`? aren't using the same DB?

Reply:
This should be refactored but refactoring ConversationsService etc. was not in scope